crates/sui-graphql-e2e-tests/tests/transactions/system.exp

```diff
@@ -253,7 +253,7 @@ Response: {
     "transactionBlockConnection": {
       "nodes": [
         {
-          "digest": "J2gfeoDEySDWfUEdboMehk45L9ndHHFv3nLF16UWXxej",
+          "digest": "CHyKA8xirNCGtcDMLJcxBZsxGyAA7DNrzDEVodxZzNn",
```

Comment:
Hi @amnn , can you please take a look if updating these digest is correct? The old one didn't include the ConsensusOutputDigest, and since we are including it, the digest here needs to change.